frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/contextmenu/ShowSelectedInGitLogAction.java

```diff
@@ -89,25 +83,12 @@ public void actionPerformed(AnActionEvent anActionEvent) {
 
   @ContinuesInBackground
   private void jumpToRevisionUnderProgress(Project project, Hash hash) {
-    new Task.Backgroundable(project, getString("action.GitMachete.ShowSelectedInGitLogAction.task-title")) {
-      @Override
-      public void run(ProgressIndicator indicator) {
-        try {
-          val logUi = VcsProjectLog.getInstance(project).getMainLogUi();
-          if (logUi == null) {
-            LOG.error("Main log ui is null");
-            return;
-          }
-          logUi.getVcsLog().jumpToReference(hash.asString()).get();
-        } catch (CancellationException | InterruptedException ignored) {} catch (ExecutionException e) {
-          String msg = "Error while showing branch in Git log";
-          if (e.getMessage() != null) {
-            msg = e.getMessage();
-          }
-          LOG.error(msg);
-        }
-      }
-    }.queue();
+    val logUi = VcsProjectLog.getInstance(project).getMainLogUi();
+    if (logUi == null) {
+      LOG.error("Main VCS Log UI is null");
+      return;
+    }
+    logUi.getVcsLog().jumpToReference(hash.asString());
```

Comment:
This returns a Java `Future`. Still, we're free to ignore (fire-and-forget) that Future, just as we effectively ignore most `Task.Backgroundable`s after `queue()`.

Reply:
no exception handling needed here? 🤔

Reply:
I don't think it's needed TBH. It's pretty hard for exception to occur here (I've checked, and passing `lolxd` results in just a tooltip; passing `null` crashes the flow before the `Future` starts). Even if it happens... it's gonna most likely be reported to the user (who hopefully reports it to us, in turn)

Reply:
ok,thx